dynawo/sources/Models/Modelica/Dynawo/Examples/DynaFlow/IEEE14/TestCases/IEEE14CLA.mo

```diff
@@ -0,0 +1,220 @@
+  Gen3.switchOffSignal1.value = false;
+  Gen3.switchOffSignal2.value = false;
+  Gen3.switchOffSignal3.value = false;
+  Gen6.switchOffSignal1.value = false;
+  Gen6.switchOffSignal2.value = false;
+  Gen6.switchOffSignal3.value = false;
+  Gen8.switchOffSignal1.value = false;
+  Gen8.switchOffSignal2.value = false;
+  Gen8.switchOffSignal3.value = false;
+  Load2.switchOffSignal1.value = false;
+  Load2.switchOffSignal2.value = false;
+  Load3.switchOffSignal1.value = false;
+  Load3.switchOffSignal2.value = false;
+  Load4.switchOffSignal1.value = false;
+  Load4.switchOffSignal2.value = false;
+  Load5.switchOffSignal1.value = false;
+  Load5.switchOffSignal2.value = false;
+  Load6.switchOffSignal1.value = false;
+  Load6.switchOffSignal2.value = false;
+  Load9.switchOffSignal1.value = false;
+  Load9.switchOffSignal2.value = false;
+  Load10.switchOffSignal1.value = false;
+  Load10.switchOffSignal2.value = false;
+  Load11.switchOffSignal1.value = false;
+  Load11.switchOffSignal2.value = false;
+  Load12.switchOffSignal1.value = false;
+  Load12.switchOffSignal2.value = false;
+  Load13.switchOffSignal1.value = false;
+  Load13.switchOffSignal2.value = false;
+  Load14.switchOffSignal1.value = false;
+  Load14.switchOffSignal2.value = false;
+  LineB10B11.switchOffSignal1.value = false;
+  LineB10B11.switchOffSignal2.value = false;
+  LineB12B13.switchOffSignal1.value = false;
+  LineB12B13.switchOffSignal2.value = false;
+  LineB13B14.switchOffSignal1.value = false;
+  LineB13B14.switchOffSignal2.value = false;
+  LineB1B5.switchOffSignal1.value = false;
+  LineB1B2.switchOffSignal1.value = false;
+  LineB2B3.switchOffSignal1.value = false;
+  LineB2B3.switchOffSignal2.value = false;
+  LineB2B4.switchOffSignal1.value = false;
+  LineB2B4.switchOffSignal2.value = false;
+  LineB2B5.switchOffSignal1.value = false;
+  LineB3B4.switchOffSignal1.value = false;
+  LineB3B4.switchOffSignal2.value = false;
+  LineB4B5.switchOffSignal1.value = false;
+  LineB4B5.switchOffSignal2.value = false;
+  LineB6B11.switchOffSignal1.value = false;
+  LineB6B11.switchOffSignal2.value = false;
+  LineB6B12.switchOffSignal1.value = false;
+  LineB6B12.switchOffSignal2.value = false;
+  LineB6B13.switchOffSignal1.value = false;
+  LineB6B13.switchOffSignal2.value = false;
+  LineB7B8.switchOffSignal1.value = false;
+  LineB7B8.switchOffSignal2.value = false;
+  LineB7B9.switchOffSignal1.value = false;
+  LineB7B9.switchOffSignal2.value = false;
+  LineB9B10.switchOffSignal1.value = false;
+  LineB9B10.switchOffSignal2.value = false;
+  LineB9B14.switchOffSignal1.value = false;
+  LineB9B14.switchOffSignal2.value = false;
+  Tfo1.switchOffSignal1.value = false;
+  Tfo1.switchOffSignal2.value = false;
+  Tfo2.switchOffSignal1.value = false;
+  Tfo2.switchOffSignal2.value = false;
+  Tfo3.switchOffSignal1.value = false;
+  Tfo3.switchOffSignal2.value = false;
+  Bank9.switchOffSignal1.value = false;
+  Bank9.switchOffSignal2.value = false;
+
+  annotation(
```

Comment:
Maybe we can add a documentation here to describe the results and explain why having such an approach (time-domain approach to calculate steady states) is important. We can describe the two cases when we change the time constants of the three CLA and explain which lines are getting disconnected (which can lead to a failure in one case).